### PR TITLE
boxed bat wings

### PR DESCRIPTION
**Pull request by bearsdotzone**

### CODE REVIEW

File: packages/greenbox-data/data/iotms.ts [hunk 1]
```diff
@@ -1282,4 +1282,11 @@ export default [
     type: "item",
     item: "Sept-Ember Censer",
   }, // boxed Sept-Ember Censer
+  {
+    id: 11657,
```
Comment (gausie) [inline packages/greenbox-data/data/iotms.ts:1286]: 11658

**Review by gausie — CHANGES_REQUESTED** [review]

File: packages/greenbox-data/data/iotms.ts (hunk 1 above)
Comment (bearsdotzone) [reply packages/greenbox-data/data/iotms.ts]: I'm curious, why is this the case? All the other iotms I looked at referred to the packaged item in their ID field.

Comment (Rinn) [reply packages/greenbox-data/data/iotms.ts]: @gausie i think you're wrong on this one, all the other entries use the packaged item id

Comment (gausie) [reply packages/greenbox-data/data/iotms.ts]: Oh

**Review by gausie — APPROVED** [review]

**Pull request merged**